docs/src/tutorial/minimal_working_compute.jl

```diff
@@ -23,9 +23,9 @@ we walk through the following steps:
 
 =#
 
-using SwiftShader_jll # hide
+using SwiftShader_jll #hide
```

Comment:
```suggestion
using SwiftShader_jll # hide
```

Reply:
I don't think the whitespace is required, see [the Documenter docs](https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@example-block). Also [the deployed page](https://juliagpu.github.io/Vulkan.jl/dev/tutorial/minimal_working_compute/) shows the intended behavior, I don't know why it wouldn't have worked for you.

Reply:
I see that the stable version of the docs indeed shows statements with `# hide`, but the dev one does not. I am not sure why, I'll tag a new version after this is merged and we'll see if the issue still occurs in stable docs.